apps/docs/pages/guides/integrations/snaplet.mdx

```diff
@@ -0,0 +1,151 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'snaplet',
+  title: 'Snaplet',
+  description:
+    'Get started with Supabase and Snaplet, a developer tool for working with safe, versioned, up-to-date production-like data',
+}
+
+This guide will help you get a development Supabase instance set up using [Snaplet](https://www.snaplet.dev/). In 15 minutes or less, you will create a snapshot of your production Supabase instance and restore it to a development environment you can safely code against.
+
+<Admonition type="note">
+If you experience any issues with getting Supabase working with Snaplet, or with this guide, feel free to chat with the Snaplet team on [Discord](https://app.snaplet.dev/chat).
+</Admonition>
+
+<div>
+
+<img src="/docs/img/guides/integrations/snaplet/snappy-holding-supabase-logo.svg/" alt="Supabase connection string" width="30%" style={{ margin: "auto" }} />
+
+</div>
+
+### Things you'll need before you begin: [#prerequisite]
+
+1. **A production Supabase project’s connection string:** This can be found in Supabase via **Organization > All Projects > Your Project > Project Settings > Connection Pooling > Connection String**. We’ll refer to this in the guide as your “source database” - where your data comes from.
+
+![Supabase connection string](/docs/img/guides/integrations/snaplet/connection_string.webp)
+
+2. A development Supabase project’s connection string: Same steps as above, but a different project/environment. We’ll refer to this in the guide as your “target database” - where we’re restoring the snapshot of the source database to. If you haven’t got this set up yet, we’ll take you through the process of setting this up.
+
+> **Highly Recommended:** Create a read-only role for your connection string. For more info on how to create a read-only role across all schemas, you can check out the [Snaplet docs](https://docs.snaplet.dev/guides/postgresql#create-a-read-only-role).
```

Comment:
Can you convert blockquotes to [admonitions](https://github.com/supabase/supabase/blob/6e1620a20adc22ae3f32ca67a5931f4da4d93344/packages/ui/src/components/Admonition/index.tsx#L5)?